Makefile

```diff
@@ -56,75 +57,54 @@ ifneq ($(DEPLOY_REGION),)
 	$(eval LIMIT=$(LIMIT):&region_$(DEPLOY_REGION))
 endif
 
-	cd ansible && $(ENV) ansible-playbook \
-		--limit="$(LIMIT)" \
-		-e ansible_python_interpreter=/usr/bin/python3 \
-		-e package=$(PACKAGE) \
-		-e env=$(DEPLOY_ENV) \
-		-e "@$(CONFIG_OUTPUT_DIR)/$(DEPLOY_ENV).yml" \
-		-e downtime=$(DEPLOY_DOWNTIME) \
-		-e db_version=$(DEPLOY_DB_VERSION) \
-		-e rolling_update="${ROLLING_UPDATE}" \
-		deploy.yml
+ansible/deploy.yml:	ANSIBLE_EXTRA_VARS=" \
+	-e package=$(PACKAGE) \
+	-e downtime=$(DEPLOY_DOWNTIME) \
+	-e rolling_update=${ROLLING_UPDATE} \
```

Comment:
i'm not sure if this can work without quotes, also why it does not use make's $() but rather ${} which is a bash var